src/models/ITestResult.ts

```diff
@@ -88,6 +88,13 @@ export interface TestType {
   modificationTypeUsed?: string | null;
   particulateTrapFitted?: string | null;
   fuelType?: string | null;
+  centralDocs?: CentralDocs;
+}
+
+export interface CentralDocs {
```

Comment:
How does our downstream process look with this model?

Are we going to just check for this entire object existing first, then examine if `issueRequired` is truthy?

We'll need to be careful to avoid something like `if(testResult.centralDocs.issueRequired)` or we could cause reference errors

Reply:
is that not overcome with `if(testResult?.centralDocs?.issueRequired)`

Reply:
Yep, should be fine - just something we need to be considering downstream of this - that this object is not always guaranteed to exist

Reply:
We want to check that the central docs object is present which should be covered by this line !testType?.centralDocs and then the Joi validation will enforce the issue required  field to be present and apply additional validation if the other 2 fields are present